test/runtests.jl

```diff
@@ -45,7 +45,11 @@ for (M, f, arity) in DiffRules.diffrules(; filter_modules=nothing)
         derivs = DiffRules.diffrule(M, f, :foo, :bar)
         @eval begin
             let
-                foo, bar = rand(1:10), rand()
+                if "mod" == string($M.$f)
+                    foo, bar = rand() + 3, rand() + 7 # make sure x/y is not integer
```

Comment:
Isn't `mod(foo, bar)` always `foo`? This seems a bit too simple maybe.

Reply:
Maybe just use @mcabbot's suggestion?

Reply:
Hmmm you are right.